src/components/Search/Search.module.scss

```diff
@@ -1,5 +1,5 @@
 .container {
-  height: 100vh;
+  height: calc(100vh - var(--header-height) - var(--footer-height));
```

Comment:
hmm, cant be simple like 100% ?

Reply:
It didn't work. I've decided to use solution described here https://css-tricks.com/quick-css-trick-how-to-center-an-object-exactly-in-the-center/
I've also done a small change to form's styles. This form will probably be rewritten in the future anyway coz I'll extend it with more inputs